packages/http-client-csharp/emitter/src/lib/client-model-builder.ts

```diff
@@ -28,9 +27,6 @@ import { fromSdkServiceMethod, getParameterDefaultValue } from "./operation-conv
 import { processServiceAuthentication } from "./service-authentication.js";
 
 export function createModel(sdkContext: SdkContext<NetEmitterOptions>): CodeModel {
-  // initialize tcgc model
-  if (!sdkContext.operationModelsMap) getAllModels(sdkContext);
```

Comment:
I guess now there is no lazy initialization in latest version of TCGC. I could be wrong.

Reply:
I am taliking about https://github.com/Azure/typespec-azure/blob/2605f0b14e22476a21b6311ad37d56bc5947d7c7/packages/typespec-client-generator-core/src/decorators.ts#L665

Reply:
According to chenjie, everything is ready in TCGC once we create the context by `createSdkContext` and then everything is cached in the context for us to use.
We do not need to call something to intiialize.
Also after this part, we have a `navigateModels` fucntion call below, which calls the `getAllModels` as well, even if it needs to call something to invoke the initialization, we are already doing it.

And I have to remove this line because in a TCGC's version update (not released yet), this `operationModelsMap`, as an internal cache, has been removed, therefore I have to remove it.